vars/dockerExecute.groovy

```diff
@@ -67,8 +106,20 @@ void call(Map parameters = [:], body) {
             if (executeInsideDocker && config.dockerImage) {
                 def image = docker.image(config.dockerImage)
                 image.pull()
-                image.inside(getDockerOptions(config.dockerEnvVars, config.dockerVolumeBind, config.dockerOptions)) {
-                    body()
+                if (!config.sidecarImage) {
```

Comment:
avoid negation to improve readability

Reply:
left like it is to keep more important option first